pkg/webui/console/views/gateway-overview/index.js

```diff
@@ -0,0 +1,140 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import bind from 'autobind-decorator'
+import { connect } from 'react-redux'
+import { Container, Col, Row } from 'react-grid-system'
+
+import sharedMessages from '../../../lib/shared-messages'
+import IntlHelmet from '../../../lib/components/intl-helmet'
+import DateTime from '../../../lib/components/date-time'
+import DataSheet from '../../../components/data-sheet'
+import GatewayStatistics from '../../containers/gateway-statistics'
+import GatewayEvents from '../../containers/gateway-events'
+import Tag from '../../../components/tag'
+
+import {
+  gatewaySelector,
+} from '../../store/selectors/gateway'
+
+import style from './gateway-overview.styl'
+
+@connect(function (state, props) {
+  const gtwId = props.match.params.gtwId
+
+  return {
+    gtwId,
+    gateway: gatewaySelector(state, props),
+  }
+})
+@bind
+export default class GatewayOverview extends React.Component {
+
+  get gatewayInfo () {
+    const { gtwId, gateway } = this.props
+    const {
+      ids,
+      name,
+      description,
+      created_at,
+      updated_at,
+      frequency_plan_id,
+      gateway_server_address,
+    } = gateway
+
+    const sheetData = [
+      {
+        header: sharedMessages.generalInformation,
+        items: [
+          {
+            key: sharedMessages.gatewayID,
+            value: gtwId,
+            type: 'code',
+            sensitive: false,
+          },
+          {
+            key: sharedMessages.gatewayEUI,
+            value: ids.eui,
+            type: 'code',
+            sensitive: false,
+          },
+          {
+            key: sharedMessages.gatewayName,
```

Comment:
I don't think we need to show the name here, we don't do it as well for apps devices. In the title section should be enough.